README.md

```diff
@@ -1,2 +1,9 @@
-# eq-questionnaire-runner-deploy-image
+# EQ Questionnaire Runner Deploy Image
 The Dockerfile used to build the EQ Questionnaire Runner base image.
+
+## Building
+The image can be built using:
+
+```sh
+docker build -t eq-questionnaire-runner-deploy-image:latest
+```
```

Comment:
There isn't anything specific to questionnaire runner in this image. We should consider naming it `eq-app-deploy-image`.